### ISSUE TITLE
Leaflet Error

### ISSUE THREAD

**Issue by sohail288** [CompostAppTeam/CompostApp#9]
@reganking 

Hey regan,  we got this error trying to render the app.  We will apply local fixes to get our stuff working, but if you can fix this, we will just stash pull and pop and go on our ways.

`Uncaught Error: Couldn't autodetect L.Icon.Default.imagePath, set it manually._getIconUrl @ leaflet-src.js:3415_createIcon @ leaflet-src.js:3320createIcon @ leaflet-src.js:3312_initIcon @ leaflet-src.js:3551onAdd @ leaflet-src.js:3471_layerAdd @ leaflet-src.js:2225addLayer @ leaflet-src.js:1683addTo @ leaflet-src.js:3481(anonymous function) @ (program):17DOMRange._insertNodeWithHooks @ domrange.js:94DOMRange._insert @ domrange.js:63DOMRange.attach @ domrange.js:140Blaze.render @ view.js:640Template.body.renderToDocument @ templating.js:67maybeReady @ startup_client.js:26loadingCompleted @ startup_client.js:38

`

**Comment by reganking** [comment]
Do you have leaflet and react-leaflet in node_modules? Is the error halting meteor? 
Try dropping in right after mapid element in main.html following:
<script src="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.js"></script>

**Comment by sohail288** [comment]
Yeah, both the leaflet and react-leaflet are present in node_modules

**Comment by sohail288** [comment]
we got rid of it from the html and are side stepping around it for now

**Comment by reganking** [comment]
ok ... my markup following: isn't showing so here: `<script src="http://cdn.leafletjs.com/leaflet/v0.7.7/leaflet.js"></script>`

**Comment by reganking** [comment]
ok, sidestep good

**Comment by reganking** [comment]
Have fix from https://github.com/Leaflet/Leaflet/issues/766
Will apply for next commit later:
L.Icon.Default.imagePath = 'path-to-your-leaflet-images-folder';

**Comment by sohail288** [comment]
added cdn link right after head element and it got rid of the error

**Comment by sohail288** [comment]
okay for now i'll keep it this way

**Comment by sohail288** [comment]
i will not commit it

**Comment by reganking** [comment]
OK, I'll take it ;)
